site/src/pages/ManagementSettingsPage/OrganizationProvisionersPage.tsx

```diff
@@ -16,13 +16,10 @@ const OrganizationProvisionersPage: FC = () => {
 	};
 	const { organizations } = useManagementSettings();
 	const { entitlements } = useDashboard();
-
 	const { metadata } = useEmbeddedMetadata();
 	const buildInfoQuery = useQuery(buildInfo(metadata["build-info"]));
 
-	const organization = organizations
-		? getOrganizationByName(organizations, organizationName)
-		: undefined;
+	const organization = organizations?.find((o) => o.name === organizationName);
```

Comment:
oh, this should probably just be using the organization specified by `useManagementSettings`